Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -379,4 +381,17 @@ CFTypeID CGFontGetTypeID() {
 // So to prevent potentially copying multiple times, save a reference to the data
 CFDataRef _CGFontGetData(CGFontRef font) {
     return font ? font->_data.get() : nullptr;
+}
+
+bool _CGFontGetGlyphsForCharacters(CGFontRef font, const char* characters, size_t count, CGGlyph* glyphs) {
+    if (!font || !characters || !glyphs || count == 0) {
+        return false;
+    }
+
+    std::vector<uint32_t> chars(characters, characters + count);
+    return SUCCEEDED(font->_dwriteFontFace->GetGlyphIndices(chars.data(), count, glyphs));
+}
+
+HRESULT _CGFontGetDWriteFontFace(CGFontRef font, IDWriteFontFace** outFace) {
+    return font ? font->_dwriteFontFace.CopyTo(outFace) : E_POINTER;
```

Comment:
Please update the usage across the file.
It would be a better choice going forward.



---
In reply to: [91551180](https://github.com/Microsoft/WinObjC/pull/1516#discussion_r91551180) [](ancestors = 91551180)